checks/runchecks.go

```diff
@@ -11,15 +12,60 @@ import (
 	"github.com/flanksource/commons/logger"
 )
 
-func RunChecks(ctx *context.Context) []*pkg.CheckResult {
+// A list of check types that are permanently disabled.
+var disabledChecks map[string]struct{}
+
+func getDisabledChecks(ctx *context.Context) (map[string]struct{}, error) {
+	if disabledChecks != nil {
+		return disabledChecks, nil
+	}
+
+	result := make(map[string]struct{})
+	if ctx.DB() == nil {
+		return result, nil
+	}
+
+	rows, err := ctx.DB().Raw("SELECT value FROM properties WHERE name = 'check'").Rows()
```

Comment:
I think the format should be: name=check.<type>.disabled, value = true 
We might also want to populate the properties table from a `canary-checker.properties`  file